src/main/kotlin/org/wycliffeassociates/otter/jvm/app/widgets/resourcecard/view/ResourceListView.kt

```diff
@@ -0,0 +1,21 @@
+package org.wycliffeassociates.otter.jvm.app.widgets.resourcecard.view
+
+import javafx.collections.ObservableList
+import javafx.scene.control.ListView
+import javafx.scene.layout.Priority
+import org.wycliffeassociates.otter.jvm.app.widgets.resourcecard.model.ResourceGroupCardItem
+import org.wycliffeassociates.otter.jvm.app.widgets.resourcecard.styles.ResourceListStyles
+import tornadofx.*
+
+class ResourceListView(items: ObservableList<ResourceGroupCardItem>): ListView<ResourceGroupCardItem>(items) {
+    init {
+        vgrow = Priority.ALWAYS // This needs to be here
```

Comment:
can this go in the stylesheet?

Reply:
No, vgrow is a property that cannot be set in the stylesheet.

Reply:
Ahh, thanks!